ChangeLog

```diff
@@ -62,6 +62,9 @@ Changes
      mbedtls_ssl_get_record_expansion() in case of ChachaPoly ciphersuites,
      or CBC ciphersuites in (D)TLS versions 1.1 or higher. Fixes #1913, #1914.
    * Add support for buffering of out-of-order handshake messages.
+   * Add calls to platform setup and teardown functions to all the example
+     programs for which this is relevant. This change enables the example
```

Comment:
I would rephrase the second sentence, as now it suggests that, in short, "this change enables the example programs". How about "This change enables the utilization of plaform-specific cryptographic features in example programs."?

Reply:
I agree the sentence is not nicely worded. AFAIK it would technically be correct to write "...enables the example programs to utilize...", but it is rather informal. I will rephrase the sentence.